apps/backend/scripts/clickhouse-migrations.ts

```diff
@@ -0,0 +1,47 @@
+import { clickhouseAdminClient } from "@/lib/clickhouse";
+import { getEnvVariable } from "@stackframe/stack-shared/dist/utils/env";
+
+export async function runClickhouseMigrations() {
+  console.log("[Clickhouse] Running Clickhouse migrations...");
+  const client = clickhouseAdminClient;
+  const clickhouseExternalPassword = getEnvVariable("STACK_CLICKHOUSE_EXTERNAL_PASSWORD");
+  await client.exec({
+    query: "CREATE USER IF NOT EXISTS limited_user IDENTIFIED WITH plaintext_password BY {clickhouseExternalPassword:String}",
+    query_params: { clickhouseExternalPassword },
+  });
+  // todo: create migration files
+  await client.exec({ query: EVENTS_TABLE_BASE_SQL });
+  await client.exec({
```

Comment:
somewhere here we probably want to set

```
users_without_row_policies_can_read_rows
```

maybe some more from here? https://clickhouse.com/docs/operations/server-configuration-parameters/settings#access_control_improvements

Reply:
do we need to set custom_settings_prefixes as well for it?